src/main/java/service/GitLabService.java

```diff
@@ -472,10 +459,20 @@ private MergeRequest waitForMrReady(String gitlabEventUUID, MergeRequest mr) {
 		return mr;
 	}
 
+	private MergeRequest getFreshMrObject(String gitlabEventUUID, MergeRequest mr) {
```

Comment:
I am not sure about this name `getFreshMrObject(...)`.

* The `Object` suffix does not match with other method names.
* And the `Fresh`. Everything that comes from the server is "Fresh"

We already have a `getMr(...)`  but it has the difference that it return null in the catch bloc.

* maybe `getMr(...)` (because of the different parameter type it can work).
* maybe `getExistingMr(...)` 
* maybe something else, I am not sure.